assets/src/styles/blocks/TableEditor.scss

```diff
@@ -1,3 +0,0 @@
-.edit-post-visual-editor {
```

Comment:
This was moved into `editorOverrides` in master theme https://github.com/greenpeace/planet4-master-theme/pull/1337/files#diff-bd9a4976ecbe15aad4890e46755af5d72a7c44d47f57d861c60207b7bfbf2cccR25